### PR TITLE
core: ketcher-related fixes: #465, #529, #538, #562, #564, #563, #571

### PR DESCRIPTION
**Pull request by even1024**
simple objects rendering, dearomatize fix, multiple ket-format fixes.

### CODE REVIEW

File: core/render2d/src/render_internal.cpp [hunk 1]
```diff
@@ -2010,6 +2010,61 @@ void MoleculeRenderInternal::_renderBonds()
 
 void MoleculeRenderInternal::_renderMeta()
 {
+    auto& md = _mol->metaData();
```
Comment (mkviatkovskii) [inline core/render2d/src/render_internal.cpp]: const

File: core/render2d/src/render_internal.cpp [hunk 2]
```diff
@@ -2010,6 +2010,61 @@ void MoleculeRenderInternal::_renderBonds()
...
+    for (int i = 0; i < md.size(); ++i)
+    {
+        auto& simple = *md[i];
```
Comment (mkviatkovskii) [inline core/render2d/src/render_internal.cpp]: const

File: core/indigo-core/graph/src/graph.cpp [hunk 3]
```diff
@@ -804,6 +804,10 @@ void Graph::_cloneGraph_KeepIndices(const Graph& other)
         if (addVertex() != i)
             throw Error("_clone_KeepIndices: unexpected vertex index");
 
+    auto& meta = other.metaData();
```
Comment (mkviatkovskii) [inline core/indigo-core/graph/src/graph.cpp:807]: const

File: core/render2d/src/render_internal.cpp (hunk 2 above)
Comment (even1024) [reply core/render2d/src/render_internal.cpp]: ашчув

File: core/indigo-core/graph/src/graph.cpp (hunk 3 above)
Comment (even1024) [reply core/indigo-core/graph/src/graph.cpp]: fixed

**Review by mkviatkovskii — APPROVED** [review]

**Pull request merged**